src/docs/development/frameworks-mobile/react-native.mdx

```diff
@@ -63,16 +65,7 @@ yarn add @pxblue/react-native-themes
 
 ## Applying the Theme
 
-You can apply the PX Blue theme by wrapping your application in a `<ThemeProvider>` and passing in the theme object.
-
-```
-import { ThemeProvider } from '@pxblue/react-native-components';
-import * as PXBThemes from '@pxblue/react-native-themes';
-...
-<ThemeProvider theme={PXBThemes.blue}>
-    <App />
-</ThemeProvider>
-```
+You can apply the PX Blue theme by wrapping your application in a `<ThemeProvider>` and passing in the theme object. For detailed information, check [our documentation](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--theme).
```

Comment:
I like the addition of the link, but what's the reasoning behind removing the code block?

Reply:
I don't want our docs to be scattered everywhere. I rather keep them all in rn comp lib docs.

Reply:
Are you planning to remove the code blocks on the Angular and React Guides then? What will be left on doc-it?

Reply:
Actually, I take it back...maybe we should remove them and really simplify the documentation on doc-it

Reply:
PXB-1343